build.sbt

```diff
@@ -7,7 +7,9 @@ lazy val microservice = Project("uknw-auth-checker-api", file("."))
   .enablePlugins(play.sbt.PlayScala, SbtDistributablesPlugin)
   .settings(
     libraryDependencies ++= AppDependencies.compile ++ AppDependencies.test,
-    // https://www.scala-lang.org/2021/01/12/configuring-and-suppressing-warnings.html
+    libraryDependencies += "io.github.wolfendale" %% "scalacheck-gen-regexp" % "0.1.3",
```

Comment:
Please can you move these out of ehre. It should be in AppDependecies.scala

e.g. https://github.com/hmrc/uknw-auth-checker-api/blob/main/project/AppDependencies.scala

Reply:
moved to AppDependencies.scala file